charts/seed-bootstrap/charts/monitoring/templates/config.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: seed-prometheus-config
+  namespace: {{ .Release.Namespace }}
+data:
+  prometheus.yaml: |
+
+    global:
+      evaluation_interval: 1m
+      scrape_interval: 1m
+
+    scrape_configs:
```

Comment:
Can you add a scrape config for the gardenlet?

Reply:
I expanded the prometheus config so that any component running in the garden namespace with the proper annotations will also be scraped.